echo-rest/src/test/groovy/com/netflix/spinnaker/echo/events/SimpleEventTemplateSpec.groovy

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.events
+
+import spock.lang.Specification
+import spock.lang.Subject
+
+class SimpleEventTemplateSpec extends Specification {
+
+    @Subject
+    RestEventTemplateEngine restEventTemplateEngine = new SimpleEventTemplateEngine()
+    Map event = [
+            'content': [
+              'action': 'resize asg'
+            ],
+            'details': [
+              'user': 'anonymous'
+            ]
+    ]
+
+    void 'renders template with event map'() {
+        given:
+        def template = '{"myEventField":"${event}"}'
```

Comment:
I think you need to drop the quotes around `${event}` to get the unit test to pass.